scripts/code.angularjs.org-firebase/functions/index.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 const functions = require('firebase-functions');
-const gcs = require('@google-cloud/storage')();
+const {Storage} = require('@google-cloud/storage');
```

Comment:
The version of `@google-cloud/storage` that we recently upgraded to had a breaking change in its API.